pyproject.toml

```diff
@@ -43,6 +43,10 @@ dependencies = [
         "six",
         "funcsigs",
         "SymPy",
+        "coverage",
+        "sphinx",
+        "sphinx_bootstrap_theme",
+        "mock",
```

Comment:
You've added the dev dependencies to the normal dependencies, which is not needed.
Please add them as optional dependencies with:
```
[project.optional-dependencies]
dev = [
    "coverage",
    "sphinx",
    "sphinx_bootstrap_theme",
    "mock",
    "pytest",
    "flake8"
]

Reply:
OK. I hope i finally understood what you want to do with the optional dependencies. In this case calling `pip install kafe2[dev] will install kafe2 and the optional dependencies listed in the `dev` array.

Reply:
I already tested it on my machine by setting up a virtual environment and calling `pip install dist/kafe2-2.8.0.tar.gz[dev]` in there and all the optional dependencies were also installed by pip.

Reply:
I just tested this in editable mode as well. When in the kafe2 project dir `pip install -e .[dev]` installs kafe2 in editbale mode including all requirements. That's ideal for development.